src/main/java/org/opentripplanner/middleware/utils/ApiGatewayUtils.java

```diff
@@ -62,6 +64,8 @@ public static CreateApiKeyResult createApiKey(String userId, String usagePlanId)
             CreateApiKeyRequest apiKeyRequest = new CreateApiKeyRequest();
             apiKeyRequest.setSdkRequestTimeout(SDK_REQUEST_TIMEOUT);
             apiKeyRequest
+                //FIXME This may need to include stage key(s). Not sure what impact that places on the calling
+                // services though?
                 .withName(userId)
```

Comment:
Why is the name of the create API key set to the user id?

Reply:
For reference. See list of API key in AWS: https://console.aws.amazon.com/apigateway/home?region=us-east-1#/api-keys

Reply:
I need to tidy this list up!